lib/meetup_bot/constants.ex

```diff
@@ -0,0 +1,10 @@
+defmodule MeetupBot.Constants do
```

Comment:
Maybe this makes more sense in [this](https://github.com/marpo60/meetup_bot/blob/main/lib/meetup_bot/event.ex) file